### PR TITLE
new cmdlet Get-AzDiskEncryptionSetAssociatedResource

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute.Test/ScenarioTests/DiskRPTests.cs [hunk 1]
```diff
@@ -93,6 +93,11 @@ public void TestSnapshotConfigDiskAccessNetworkPolicy()
         {
             TestRunner.RunTestScript("Test-SnapshotConfigDiskAccessNetworkPolicy");
```
Comment (Sandido) [inline src/Compute/Compute.Test/ScenarioTests/DiskRPTests.cs:94]: The name of the PR needs to be more informative. Indicating the cmdlets at least.

File: src/Compute/Compute/help/Get-AzDiskEncryptionSetAssociatedResource.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DiskEncryptionSetName
+{{ Fill DiskEncryptionSetName Description }}
```
Comment (Sandido) [inline src/Compute/Compute/help/Get-AzDiskEncryptionSetAssociatedResource.md:53]: Please fill the 2 descriptions out.

**Review by Sandido — APPROVED** [review]

File: src/Compute/Compute/help/Get-AzDiskEncryptionSetAssociatedResource.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Compute.dll-Help.xml
+Module Name: Az.Compute
+schema: 2.0.0
```
Comment (hyonholee) [inline src/Compute/Compute/help/Get-AzDiskEncryptionSetAssociatedResource.md:4]: You need online version:

Check other help file.

File: src/Compute/Compute/help/Get-AzDiskEncryptionSetAssociatedResource.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Compute.dll-Help.xml
+Module Name: Az.Compute
```
Comment (hyonholee) [inline src/Compute/Compute/help/Get-AzDiskEncryptionSetAssociatedResource.md:3]: Was this file generated from the help generator?

**Pull request merged**